api/lib/models/Location.js

```diff
@@ -1,80 +0,0 @@
-import * as Chance                       from 'chance';
```

Comment:
Please put this file back and comment out everything. We would like to keep it for reference until at least the end of migration